src/DashboardService.Models.Dto/Responses/TaskResponse.cs

```diff
@@ -0,0 +1,22 @@
+using LT.DigitalOffice.DashboardService.Models.Db;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Responses;
+
+public record TaskResponse
+{
+  public Guid Id { get; set; }
+  public Guid? TaskTypeId { get; set; }
+  public Guid? PriorityId { get; set; }
+  public string Name { get; set; }
+  public string Content { get; set; }
+  public DateTime CreatedAtUtc { get; set; }
+  public DateTime DeadlineAtUtc { get; set; }
+  
+  public DbGroup Group { get; set; }
+  public DbTaskType TaskType { get; set; }
+  public DbPriority Priority { get; set; }
+  public ICollection<DbComment> Comments { get; set; }
+  public ICollection<DbChangeLog> Logs { get; set; }
```

Comment:
better return dto models

Reply:
ill do it when all ours infrastructure PR`s will be merged in develop branch. Because in that branch i haven't these DTO`s. I set TODO task for future